jstree/src/js/config.js

```diff
@@ -0,0 +1,124 @@
+jQuery.noConflict();
+(function($, PLUGIN_ID) {
+    'use strict';
+    const $form = $('.js-submit-settings');
+    const $folder = $('#select_dropdown');
+    const $text = $('#select_text');
+    const $cancelButton = $('.js-cancel-button');
+    const $root = $('.js-text-message');
+    const $viewname = $('.js-view-name');
+    let config = kintone.plugin.app.getConfig(PLUGIN_ID);
+    let buttonbool = false;
+    document.getElementById('viewmaker').onclick = function() {
+        if ($viewname === '') {
+            window.alert('Please enter a folder view name.');
+        } else {
+            buttonbool = true;
+            const app = {
+                'app': kintone.app.getId(),
+            };
+            kintone.api(kintone.api.url('/k/v1/preview/app/views', true), 'GET', app, (view) => {
+                view.app = kintone.app.getId();
+                let maxindex = -1;
+                let firstview = true;
+                for (const numb in view.views) {
+                    if ((view.views.hasOwnProperty(numb))) {
+                        if (view.views[numb].index > maxindex) {
+                            maxindex = view.views[numb].index + 1;
+                            firstview = false;
+                        }
+                    }
+                }
+                if (firstview) {
+                    maxindex = 0;
+                }
+                view.views[$viewname.val()] = {
+                    'filterCond': '',
+                    'name': $viewname.val(),
+                    'type': 'CUSTOM',
+                    'html': '<ul id="tree"></ul>',
+                    'index': maxindex,
+                };
+                kintone.api(kintone.api.url('/k/v1/preview/app/views', true), 'PUT', view, (resp) => {
+                    // success
+                    window.alert('View Created!');
+                }, function(error) {
+                    window.alert('Please enter a folder view name.');
+                    // error
+                    console.log(error);
+                });
+                // success
+            }, function(error) {
+                // error
+                console.log(error);
+            });
+        }
+    };
+    KintoneConfigHelper.getFields().then(function(resp) {
+        for (let i = 0; i < resp.length; i++) {
+            switch (resp[i].type) {
+            case 'DROP_DOWN':
+                var opt = $('<option>'); document.createElement('option');
```

Comment:
- You don't need `document.createElement('option')` anymore.
- Use `const` instead.